OSVR-Unity/Assets/OSVRUnity/src/VREye.cs

```diff
@@ -247,20 +249,31 @@ public void CreateSurfaces(uint surfaceCount)
 
                     //distortion
                     bool useDistortion = Viewer.DisplayController.DisplayConfig.DoesViewerEyeSurfaceWantDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
-                    if(useDistortion)
+                    useDistortion |= !displayController.UseRenderManager;
+
+                    if (useDistortion)
                     {
                         //@todo figure out which type of distortion to use
                         //right now, there is only one option, SurfaceRadialDistortion
                         //get distortion parameters
-                        OSVR.ClientKit.RadialDistortionParameters distortionParameters =
-                        Viewer.DisplayController.DisplayConfig.GetViewerEyeSurfaceRadialDistortion(
-                        Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
+                        OSVR.ClientKit.RadialDistortionParameters distortionParameters;
+
+                        try
+                        {
+                            distortionParameters = Viewer.DisplayController.DisplayConfig.GetViewerEyeSurfaceRadialDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
```

Comment:
I don't have a 1.3 or 1.4 to test on, but it does not look good on HDK2. I can't really tell if I'd call it better or worse than the default without testing some other scenes. Using default values might work for one set of lenses, but it's not a good generalized solution. HDK2 and other HMDs do not have radially distorted lenses, so that distortion effect isn't accurate. 

However, if it does look better than the default for 1.3 and 1.4 users, I'd be ok with merging it in given that most users are using the RenderManager path anyway. Porting RenderManager-Unity to OSX/Linux is still the best solution, as you've pointed out in another thread.

@rpavlik any idea why GetViewerEyeSurfaceRadialDistortion fails? Does it depend on RM?